sdk/runner/sdk.go

```diff
@@ -0,0 +1,192 @@
+package runnersdk
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/mesg-foundation/engine/container"
+	"github.com/mesg-foundation/engine/cosmos"
+	"github.com/mesg-foundation/engine/hash"
+	"github.com/mesg-foundation/engine/instance"
+	"github.com/mesg-foundation/engine/protobuf/api"
+	"github.com/mesg-foundation/engine/runner"
+	accountsdk "github.com/mesg-foundation/engine/sdk/account"
+	instancesdk "github.com/mesg-foundation/engine/sdk/instance"
+	servicesdk "github.com/mesg-foundation/engine/sdk/service"
+	"github.com/mesg-foundation/engine/x/xos"
+	"github.com/tendermint/tendermint/mempool"
+)
+
+// SDK is the runner sdk.
+type SDK struct {
+	cdc          *codec.Codec
+	accountSDK   *accountsdk.SDK
+	serviceSDK   servicesdk.Service
+	instanceSDK  *instancesdk.SDK
+	client       *cosmos.Client
+	container    container.Container
+	port         string
+	engineName   string
+	ipfsEndpoint string
+}
+
+// New returns the runner sdk.
+func New(cdc *codec.Codec, client *cosmos.Client, accountSDK *accountsdk.SDK, serviceSDK servicesdk.Service, instanceSDK *instancesdk.SDK, container container.Container, engineName, port, ipfsEndpoint string) *SDK {
+	sdk := &SDK{
+		cdc:          cdc,
+		container:    container,
+		accountSDK:   accountSDK,
+		serviceSDK:   serviceSDK,
+		instanceSDK:  instanceSDK,
+		client:       client,
+		port:         port,
+		engineName:   engineName,
+		ipfsEndpoint: ipfsEndpoint,
+	}
+	return sdk
+}
+
+// Create creates a new runner.
+func (s *SDK) Create(req *api.CreateRunnerRequest, accountName, accountPassword string) (*runner.Runner, error) {
+	account, err := s.accountSDK.Get(accountName)
+	if err != nil {
+		return nil, err
+	}
+	// TODO: pass account by parameters
+	accNumber, accSeq := uint64(0), uint64(0)
```

Comment:
We should use the `account.GetSequence()` like here https://github.com/mesg-foundation/engine/pull/1462